src/main/java/com/iota/iri/controllers/AddressViewModel.java

```diff
@@ -38,6 +42,30 @@ private AddressViewModel(Address hashes, Indexable hash) {
         this.hash = hash;
     }
 
+    /**
+     * Constructor for an {@link Address} set controller from an existing {@link Address} set.
+     * The resulting hash list is sorted
+     *
+     * @param tangle The tangle reference for the database to find the {@link Address} set in
+     * @param hash The hash identifier for the {@link Address} set that needs to be found
+     * @return The {@link AddressViewModel} controller generated
```

Comment:
We are returning a list of transactions (actually transaction controllers unfortunately)